src/Directory.Build.targets

```diff
@@ -92,6 +93,19 @@
        in VS 15.5 (which is what we use for Jenkins right now). -->
   <Target Name="GenerateSatelliteAssemblies" />
 
+  <Target Name="RemoveSatelliteDllsFromBuildOutputInPackage"
+          BeforeTargets="GenerateNuspec"
+          AfterTargets="_CalculateInputsOutputsForPack"
```

Comment:
`After`, or `DependsOn`? If really after, please comment why because that's a confusing relationship.

Reply:
I wanted this to trigger before GenerateNuspec which has a `DependsOn` that ends with `_CalculateInputsOutputsForPack`.  What's the right way to say "run me before this target but after its dependson?

Reply:
Technically, just BeforeTargets does that, because rule 5 is after rule 4: https://docs.microsoft.com/en-us/visualstudio/msbuild/target-build-order#determining-the-target-build-order

I'd say to also say `DependsOnTargets="$(GenerateNuspecDependsOn)"` but the fact that it adds to it in its dependencies but not in its properties makes that a bit annoying.